### ISSUE TITLE
Can't add multiple wallpapers, or drag a wallpaper/wallpapers, if i try to drag them it makes the App crash

### ISSUE THREAD

**Issue by St1nGxD** [rocksdanister/lively#1262]
i've been trying to add multiple wallpapers through the option ' add wallpaper ' but its not letting me pick more than one, when i try to drag just one wallpaper or more than that it makes the browser crash, but the application is still running the wallpaper that i have selected is still there, hope all this is fixed, thanks in advance

https://user-images.githubusercontent.com/95598517/173177764-ede214c1-ec20-423b-b577-d79e1fc28089.mp4

.

**Comment by rocksdanister** [comment]
Yes its listed in the changelog as known issue, its a WindowsAppSDK bug so will need to wait for patch:
https://github.com/microsoft/microsoft-ui-xaml/issues/7203#issuecomment-1149018604

Use filepicker (Add wallpaper -> Browse -> Select each file 1 by 1)
Multiple file selection not enabled in filepicker is a mistake, will turn it on next update.

**Comment by rocksdanister** [comment]
Multiple file selection is enabled in filepicker v2.0.2.2 update and all dragdrop operation is disabled in installer version. (its still enabled for add wallpaper dialog in store version for now.)
